src/ping/constants.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
 module.exports = {
-  PROTOCOL: '/ipfs/ping/1.0.0',
+  PROTOCOL: 'ping/1.0.0',
```

Comment:
This is breaking.

Reply:
Maybe revert the change, depreciate this constant, and add an accessor to get the actual protocol name?